robots/barr-e/python/barr_e/workloads/mind/mind_test_workload.py

```
@@ -0,0 +1,272 @@
+            k: _safe_float(cfg.get(f"initial_{k}", 0.0)) for k in self.state_keys
+        }
+
+        self.current_state = dict(self.initial_state)
+        self.target_state = dict(self.initial_state)
+
+        self._prev_start = 0
+        self._prev_reset = 0
+
+        if rsc_path:
+            try:
+                with open(rsc_path, "r", encoding="utf-8") as f:
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)